lib/decrypt.rb

```diff
@@ -0,0 +1,10 @@
+require_relative 'enigma'
+
```

Comment:
not sure if my runner file code should be encapsulated in methods, may refactor if i have time